docs(preventdefault): add info about passive listener (#36098)
mdn/content: files/en-us/web/api/event/preventdefault/index.md

{{domxref("EventTarget.dispatchEvent()")}}, without specifying
`cancelable: true` has no effect.

> If a passive listener does call preventDefault(), the user agent will do nothing other than generate a console warning.

Reviewer: ```suggestion
If a passive listener calls `preventDefault()`, nothing will happen and a console warning may be generated.
```
